aiohttp/web_fileresponse.py

```diff
@@ -35,9 +36,15 @@ class SendfileStreamWriter(StreamWriter):
     def __init__(self,
                  protocol: BaseProtocol,
                  loop: asyncio.AbstractEventLoop,
+                 fobj: IO[Any],
+                 count: int,
```

Comment:
man 2 sendfile:
`sendfile() will transfer at most 0x7ffff000 (2,147,479,552) bytes, returning the number of bytes actually transferred.  (This is true on both 32-bit and 64-bit systems.)`

I don't follow the code, but seems we have to use loop over chunks less than specified value.

Reply:
Good point

Reply:
@socketpair actually Linux accepts larger values but transmits fewer bytes than specified.

See https://bugs.python.org/issue35179